subprojects/launcher/src/main/java/org/gradle/launcher/cli/CommandLineActionFactory.java

```diff
@@ -65,6 +68,8 @@
     private static final String HELP = "h";
     private static final String VERSION = "v";
 
+    private final ScriptFileResolver scriptFileResolver = DefaultScriptFileResolver.forScriptingLanguages(new DefaultScriptingLanguages());
```

Comment:
:x: Clients don't need to know about `DefaultScriptingLanguages` and how to instantiate it. Please find my suggested fix at ab94b245695154dcf4220b7e3239e18f8013e279